remedy.lic

```diff
@@ -219,31 +179,55 @@ class Remedy
       get_item('pestle')
       commmand = "crush my #{@noun} with my pestle"
     when 'That tool does not seem suitable for that task.'
+      analyze_crush
+    end
+    waitrt?
+    crush(command)
```

Comment:
The `analyze_crush` function calls `crush` at the end. Will this result in two calls to `crush` in the cases where `analyze_crush` is called?